examples/toy_model_mistis/toy_mistis_3_analysis.ipynb

```diff
@@ -965,21 +965,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.1"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.15"
   }
```

Comment:
I think this change from python 3 to 2 should not matter, right? These notebooks should be compatible with both anyway...

Reply:
This changes to whatever you used the last time you saved the notebooks. I was in Py 2.7 when I made the updates here. But these notebooks are run in our test suite, so as long as tests pass, they must work with both Python 2 and 3!